platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -306,32 +306,63 @@ the platform team include the following:
-
-And because a platform's success also depends on its users, platforms should
-also measure users' experiences, including:
-
-- Time for a new contributor to submit their first (or 10th!) PR
-- Number of contributions from users, e.g. Backstage plugins, Crossplane
-  compositions
-- Number of projects which diverge from "golden paths" and provided capabilities
-- Number of tickets opened related to platform capabilities
-- Developer satisfaction in the enterprise
-    - Net Promoter Score (NPS)
-    - SPACE framework: https://queue.acm.org/detail.cfm?id=3454124
+Enterprises will want to measure whether their platform initiatives are
+delivering their anticipated values. Also, throughout this paper we've
+emphasized the importance of treating internal platforms as products, and good
+product management depends on quantitative and qualitative measurement of a
+product's performance. To meet these requirements, internal platform teams
+should gather user feedback and measure user activities.
+
+As with other aspects of internal platforms, though, platform teams should use
+the smallest viable effort to gather the feedback they need. We'll suggest
+metrics here but simple surveys or question and answer sessions may be most
+valuable initially.
+
+Categories of metrics that will help enterprises and platform teams understand
+the impact of their platforms include the following:
+
+1. User satisfaction and productivity
+
+The first quality sought by many platforms is to improve user experience in order
+to increase productivity. Metrics that reflect user satisfaction and
+productivity include the following:
+
+- Active users and resources: number of capabilities provisioned and used from the platform
+- "Net Promoter Score" (NPS) or other surveys that measure user satisfaction with a product
+- Metrics for developer productivity such as those discussed in the SPACE framework [[3]]
+
+1. Organizational efficiency
+
+Another benefit sought from many platforms is to efficiently provide common
+needs to a large user base. This is often achieved by enabling user self-service
+and reducing manual steps and required human intervention while implementing
+policies to guarantee safety and compliance. To measure the efficiency of a
+platform in reducing common work, consider measures such as these:
+
+- Latency from request to fulfillment of a service or capability, such as a database or test environment
+- Latency to build and deploy a brand new service into production
+- Time for a new user to submit their first code changes to their product
+
+1. Product and feature delivery
+
+The ultimate objective of internal platforms is to deliver business value to
+customers faster, so measuring impact on a business's own product and feature
+releases demonstrates that the objectives of the platform are being met. The
+DevOps Research and Assessment (DORA) institute at Google suggests [[4]]
+tracking the following metrics:
+
+- Deployment frequency
```

Comment:
here if we want to be more specific we can give examples such as 

- decrease in time from commit to deploy
- decrease in MTTR 
- decrease in change fail rate 
- increase in deployments

Reply:
Well you won't know if they increase or decrease till you measure them!